### ISSUE TITLE
[UnexpectedValueException] Failed to extract PEAR package try to use Azure

### ISSUE THREAD

**Issue by shooding** [thephpleague/flysystem#530]
Hi there,

When calling composer require league/flysystem-azure

```
[UnexpectedValueException]                                                                                                                                                      
  Failed to extract PEAR package /Users/ME/Projects/flysystem/vendor/pear-pear.php.net/XML_Util/XML_Util-1.3.0.tgz to /Users/ME/Projects/flysystem/vendor/pear-pear.  
  php.net/XML_Util. Reason: internal corruption of phar "/Users/ME/Projects/flysystem/vendor/pear-pear.php.net/XML_Util/XML_Util-1.3.0.tgz" (__HALT_COMPILER(); not found) 
```

Already added to composer.json
```
"repositories": [
        {
            "type": "pear",
            "url": "http://pear.php.net"
        }
    ]
```

It simply works on Ubuntu but not on my Mac OS X laptop.

**Comment by frankdejonge** [comment]
@shooding this seems to be an issue with php itself. A quick google showed me this was fixed in a patch version for php versions 5.4, 5.5, and 5.6. Seems the only way to fix this is to update your PHP version to a higher patch version.

**Comment by frankdejonge** [comment]
@shooding ref: https://bugs.php.net/bug.php?id=67865

**Comment by shooding** [comment]
I'm using 5.6.2 already. Looks like the bug is coming back.

**Comment by frankdejonge** [comment]
@shooding this should also be fixed in later php 5.6 version. Since Flysystem itself can't do anything about this I'm closing the issue.

**Comment by shooding** [comment]
A quick update. By using -vvv option in composer, found the issue may be related to cache.
Solved the problem after upgrading to 5.6.2 "AND" running 
```
composer clear-cache
composer update
```
